specification/apicenter/resource-manager/Microsoft.ApiCenter/stable/2024-03-01/apicenter.json

```diff
@@ -143,6 +146,147 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiCenter/deletedServices": {
```

Comment:
curious, who will be calling this?

Reply:
Customers who want to recover the service. I'm looking at APM example: https://learn.microsoft.com/en-us/azure/api-management/soft-delete#list-deleted-api-management-instances

Reply:
we will allow them to list all deleted services?

Reply:
Of course.